charts/merlin/README.md.gotmpl

```diff
@@ -55,8 +55,14 @@ $ helm uninstall merlin
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 This includes the dependencies that were installed by the chart. Note that, any PVCs created by the chart will have to be deleted manually.
 
-## Configuration
+### ReleasedVersion
+* The purpose of `.Values.releasedVersion` is to update all configs that share this tag using this 1 value in values.yaml
+`.Values.releasedVersion` is used in rendering `merlin.config` partial template and `merlin.deploymentTag` partial template
```

Comment:
Should this (and other parts of the doc in this MR) be referring to `.Values.rendered.releasedVersion` rather than `.Values.releasedVersion`?

Reply:
Yes it should, will update it!

Reply:
fixed